src/components/ProductCard.jsx

```diff
@@ -1,15 +1,56 @@
 import React from 'react';
```

Comment:
em outras partes do código está sendo feita a desconstrução do Component aqui para ser utilizado na linha 4. talvez fosse legal a gente manter esse padrão. se achar uma boa ideia, ficaria

```suggestion
import React, { Component } from 'react';
```